src/js/utils/apiClient.jsx

```diff
@@ -54,19 +54,7 @@ const handleSuccess = response => response;
 const handleError = (error) => {
   switch (error.response.status) {
     case 400:
-      Alert.error(`<div>Bad request.</br>${_.get(error, 'response.data.errorMessage', '')}</div></br>
-        <div>
-          ${_.map(error.response.data.data, (item) => {
-    let message = '';
-    if (item.defaultMessage) {
-      message += `${item.defaultMessage}</br>`;
-    }
-    if (item.field && item.rejectedValue) {
-      message += `Field: "${item.field}" with value: ${item.rejectedValue}</br>`;
-    }
-    return `<span>${message}</span>`;
-  })}
-        </div>`);
+      Alert.error(`Bad Request.</br> ${_.map(_.get(error, 'response.data.errorMessages', ''), errorMessage => `<span>${errorMessage}</span>`)}`);
```

Comment:
The span is probably going to look ugly when we return multiple items in `errorMessages`. Change that to be a div at the very least. But I think a list would be better here.